src/tactic/set.lean

```diff
@@ -0,0 +1,45 @@
+/-
```

Comment:
My comments on `tactic/set.lean` reduce it to a one line tactic plus documentation. I don't think it needs its own file -- it can be added at the bottom of `tactic/tauto.lean`.

Reply:
Done.